charts/nri-metadata-injection/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic metadata injection webhook.
 name: nri-metadata-injection
-version: 1.5.4
-appVersion: 1.5.0
+version: 1.6.0
```

Comment:
should bump the major because of the breaking change?

Reply:
Good point, done!